templates/default/systemd.erb

```diff
@@ -4,10 +4,11 @@ After=network.target
 Wants=network.target
 
 [Service]
+EnvironmentFile=-/etc/sysconfig/<%= @name %>
```

Comment:
Instead of using EnvironmentFile, you can use the Environment directive and not depend on any files at all